usage/app/controllers/UsageApi.scala

```diff
@@ -204,19 +204,25 @@ class UsageApi(auth: Authentication, usageTable: UsageTable, usageGroup: UsageGr
   }}
 
   def deleteUsages(mediaId: String) = auth.async {
-    usageTable.queryByImageId(mediaId).map(usages => {
-      usages.foreach(usageTable.deleteRecord)
-    }).recover{
-      case error: BadInputException =>
-        logger.warn("UsageApi returned an error.", error)
-        respondError(BadRequest, "image-usage-delete-failed", error.getMessage)
-      case error: Exception =>
-        logger.error("UsageApi returned an error.", error)
-        respondError(InternalServerError, "image-usage-delete-failed", error.getMessage)
-    }
+      if (mediaId.isEmpty) {
+       Future.successful(respondError(BadRequest, "image-usage-delete-failed", "Congratulations for calling delete without a mediaId!"))
+     }
+     else{
+      usageTable.queryByImageId(mediaId).map(usages => {
+        usages.foreach(usageTable.deleteRecord)
+        val updateMessage = UpdateMessage(subject = "delete-usages", id = Some(mediaId))
+        notifications.publish(updateMessage)
+        Ok
+      }).recover{
+        case error: BadInputException =>
+          logger.warn("UsageApi returned an error.", error)
+          respondError(BadRequest, "image-usage-delete-failed", error.getMessage)
+        case error: Exception =>
+          logger.error("UsageApi returned an error.", error)
+          respondError(InternalServerError, "image-usage-delete-failed", error.getMessage)
+      }
+
 
-    val updateMessage = UpdateMessage(subject = "delete-usages", id = Some(mediaId))
```

Comment:
Is the removal of `updateMessage` from this file a part of the migration work?

Reply:
it's just github being fussy about the diff https://github.com/guardian/grid/pull/3176/files#diff-633bb59916ca5d633a22a47735c725b8fcc45f2f1f6ae67406346a80037c2755R213